ui/apps/platform/src/Containers/VulnMgmt/widgets/TopRiskiestEntities.js

```diff
@@ -236,23 +252,34 @@ const getQueryBySelectedEntity = (entityType) => {
     }
 };
 
-const getEntitiesByContext = (entityContext) => {
+const getEntitiesByContext = (entityContext, showVmUpdates) => {
     const entities = [];
-    if (entityContext === {} || !entityContext[entityTypes.COMPONENT]) {
-        entities.push({ label: 'Top Riskiest Components', value: entityTypes.COMPONENT });
+    if (!showVmUpdates && (entityContext === {} || !entityContext[entityTypes.COMPONENT])) {
+        entities.push({
+            label: 'Top Riskiest Components',
+            value: entityTypes.COMPONENT,
+        });
```

Comment:
I'll modify this to address ROX-11633 later

Reply:
FYI -- i ended up adding it to this PR felt like it would be helpful to minimize merge conflicts